GoogleSignIn/Sources/GIDHTTPFetcher/Implementations/GIDHTTPFetcher.m

```diff
@@ -0,0 +1,32 @@
+#import "GoogleSignIn/Sources/GIDHTTPFetcher/Implementations/GIDHTTPFetcher.h"
+
+#ifdef SWIFT_PACKAGE
+@import GTMAppAuth;
```

Comment:
Note that we should be importing GTMSessionFetcher here, not GTMAppAuth.  We're currently getting GTMSessionFetcher transitively.